Wikipedia/Code/WMFCVLSection.h

```diff
@@ -0,0 +1,40 @@
+#import <Foundation/Foundation.h>
+
+@class WMFCVLColumn;
+@class WMFCVLAttributes;
+@class WMFCVLInvalidationContext;
+
+/*!
+ @class        WMFCVLSection
+ @abstract     A WMFCVLSection represents a section within a column of a WMFColumnarCollectionViewLayout. It handles adjusting the size and offset of items within the section.
+ @discussion   ...
+ */
+@interface WMFCVLSection : NSObject
+
+@property (nonatomic, readonly) NSInteger index;
+@property (nonatomic) CGRect frame;
+@property (nonatomic, weak, nullable) WMFCVLColumn *column;
```

Comment:
Would that be better? That way you could remove the state? Or is that expensive? Is it tracked via indexSets?